libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -3350,6 +3350,20 @@ private static IEnumerable<object> Flatten(IEnumerable<object> list, int dept)
                     },
                     ReturnType.Number,
                     expr => ValidateOrder(expr, null, ReturnType.Array | ReturnType.String, ReturnType.Object)),
+                StringTransform(
+                                ExpressionType.SentenceCase,
+                                args =>
+                                {
+                                    var inputStr = (string)args[0];
+                                    if (string.IsNullOrEmpty(inputStr))
+                                    {
+                                        return string.Empty;
+                                    }
+                                    else
+                                    {
+                                        return inputStr.Substring(0, 1).ToUpper() + inputStr.Substring(1);
```

Comment:
>ToUpper() [](start = 72, length = 9)

Eventually we will need a locale here, but we can defer to when we update locale support across the board.